lighthouse-core/runner.js

```diff
@@ -107,6 +107,12 @@ class Runner {
       // Entering: conclusion of the lighthouse result object
       const lighthouseVersion = require('../package.json').version;
 
+      // Use version from gathering stage. In case accessibility category doesn't run,
+      // fallback to package json so the report will always have some version.
```

Comment:
Do we really care about including info on a library when we didn't run it? The larger question is probably should we be generalizing this to the equivalent of `chrome://credits/` for all our libraries in the lhr but then selectively show in the report, but I also don't feel strongly.